tests/exec-steps.sh

```diff
@@ -20,6 +20,8 @@ for STEP in /var/lib/tests/step-*.sh; do
         mkdir -p /usr/share/git-core/templates/hooks
         rm -rf /usr/share/git-core/templates/hooks/*
     fi
+
+    rm -rf ~/test*
```

Comment:
Is this related to the whitespace in all the test names? If so, please revert for now.

Reply:
No it isn't, these are some artefacts from tests which write to the home directory, as I found :-) 
I 'd leave it :-)

Reply:
OK